### PR TITLE
Check if key is "model" in del_artifact

### PR DESCRIPTION
**Pull request by nhatsmrt**
Forgot to check this... User might accidentally delete model via del_artifact.

### CODE REVIEW

File: client/verta/verta/_registry/modelversion.py [hunk 1]
```diff
@@ -208,6 +208,9 @@ def get_artifact(self, key):
         return artifact_stream
 
     def del_artifact(self, key):
+        if key == "model":
+            raise ValueError("the key \"model\" is reserved for model; consider using del_model() instead")
```
Comment (convoliution) [inline client/verta/verta/_registry/modelversion.py]: I think this sounds a little more straightforward than "key is reserved" for deletion
```suggestion
            raise ValueError("model can't be deleted through del_artifact(); consider using del_model() instead")
```

**Review by convoliution — CHANGES_REQUESTED** [review]
Small wording suggestion

File: client/verta/verta/_registry/modelversion.py (hunk 1 above)
Comment (nhatsmrt) [reply client/verta/verta/_registry/modelversion.py]: Fixed.

**Review by convoliution — APPROVED** [review]
Good catch!

**Pull request merged**